### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by biodranik**
Please help translate it to your language before it's merged.

Propose suggestions or just write translations in the comments. Note that Android release notes should be less or equal to 500 characters.

### CODE REVIEW

File: android/src/fdroid/play/listings/ca/release-notes.txt [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-• Fixed pedestrian ETA for big slopes
-• Better car routing
-• Improved search ranking
-• Share button has been moved to the Place Page
-• Updated many translations
-• Added new shops and event venues types
-• Improved water tunnels and bridges rendering
-• More detailed railways
-• Added boundary=protected_area
+• New OpenStreetMap data as of August 14
```
Comment (BennyBeat) [inline android/src/fdroid/play/listings/ca/release-notes.txt:1]:
```suggestion
• Dades de l'OpenStreetMap actualitzades al 14 d'agost
```

File: android/src/fdroid/play/listings/ca/release-notes.txt [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
+• Fresh Wikipedia articles
+• Fixed search for addresses in Czech and Slovakia
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia
```
Comment (BennyBeat) [inline android/src/fdroid/play/listings/ca/release-notes.txt:4]:
```suggestion
• S'ha afegit informació faltant per a algunes regions d'Argentina, Xile, Casaquistan, Tibet, USA i Rússia
```

File: android/src/fdroid/play/listings/ca/release-notes.txt [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
+• More dense isolines in some areas
+• Fixed bug on Nexus 5X
```
Comment (BennyBeat) [inline android/src/fdroid/play/listings/ca/release-notes.txt:6]:
```suggestion
• S'ha corregit un error al Nexus 5X
```

File: android/src/fdroid/play/listings/ca/release-notes.txt [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
+• Added ferry routes to the World map
```
Comment (BennyBeat) [inline android/src/fdroid/play/listings/ca/release-notes.txt:7]:
```suggestion
• S'han afegit rutes en ferri al mapa mundial
```

File: android/src/fdroid/play/listings/ca/release-notes.txt [hunk 5]
```diff
@@ -1,14 +1,13 @@
...
+• Redesigned parking icons
+• Revamped priorities system for POIs
```
Comment (BennyBeat) [inline android/src/fdroid/play/listings/ca/release-notes.txt:9]:
```suggestion
• Sistema de prioritats renovat per als PDI
```

File: android/src/fdroid/play/listings/pt-BR/release-notes.txt [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
+• Fresh Wikipedia articles
+• Fixed search for addresses in Czech and Slovakia
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia
+• More dense isolines in some areas
+• Fixed bug on Nexus 5X
+• Added ferry routes to the World map
+• Redesigned parking icons
+• Revamped priorities system for POIs
+• Many other smaller enhancements and bug fixes
+• Faster, improved search and routing
```
Comment (matheusgomesms) [inline android/src/fdroid/play/listings/pt-BR/release-notes.txt:11]:
```suggestion
• Novos dados OSM de 14/08
• Artigos da Wikipédia atualizados
• Pesquisa corrigida de endereços na Eslováquia e República Tcheca
• Adicionado isolinhas ausentes para alguns países
• Isolinhas mais densas em algumas áreas
• Correção de bug no Nexus 5X
• Adição de rotas de balsa ao mapa do mundo
• Ícones de estacionamento redesenhados
• Sistema de prioridades reformulado para POIs
• Muitas outras melhorias e correções
• Melhorias na busca e roteamento
```

File: android/src/fdroid/play/listings/pt-BR/release-notes.txt [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
 
-https://omaps.org/news
+…more: omaps.org/news
```
Comment (matheusgomesms) [inline android/src/fdroid/play/listings/pt-BR/release-notes.txt:13]:
```suggestion
...mais: omaps.org/news
```

File: android/src/fdroid/play/listings/ca/release-notes.txt [hunk 8]
```diff
@@ -1,14 +1,13 @@
...
+• Many other smaller enhancements and bug fixes
```
Comment (BennyBeat) [inline android/src/fdroid/play/listings/ca/release-notes.txt:10]:
```suggestion
• Moltes altres petites millores i correcció d'errors
```

File: android/src/fdroid/play/listings/pt-PT/release-notes.txt [hunk 9]
```diff
@@ -1,13 +1,13 @@
...
+• Fresh Wikipedia articles
+• Fixed search for addresses in Czech and Slovakia
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia
+• More dense isolines in some areas
+• Fixed bug on Nexus 5X
+• Added ferry routes to the World map
+• Redesigned parking icons
+• Revamped priorities system for POIs
+• Many other smaller enhancements and bug fixes
+• Faster, improved search and routing
```
Comment (matheusgomesms) [inline android/src/fdroid/play/listings/pt-PT/release-notes.txt:11]:
```suggestion
• Novos dados OSM de 14/08
• Artigos da Wikipédia atualizados
• Pesquisa corrigida de endereços na Eslováquia e República Checa
• Adicionado isolinhas ausentes para alguns países
• Isolinhas mais densas em algumas áreas
• Correção de bug no Nexus 5X
• Adição de rotas de balsa ao mapa do mundo
• Ícones de estacionamento redesenhados
• Sistema de prioridades reformulado para POIs
• Muitas outras melhorias e correções
• Melhorias na busca e roteamento
```

File: android/src/fdroid/play/listings/pt-PT/release-notes.txt [hunk 10]
```diff
@@ -1,13 +1,13 @@
...
 
-https://omaps.org/news
+…more: omaps.org/news
```
Comment (matheusgomesms) [inline android/src/fdroid/play/listings/pt-PT/release-notes.txt:13]:
```suggestion
...mais: omaps.org/news
```

File: android/src/fdroid/play/listings/ca/release-notes.txt [hunk 11]
```diff
@@ -1,14 +1,13 @@
...
+• Faster, improved search and routing
```
Comment (BennyBeat) [inline android/src/fdroid/play/listings/ca/release-notes.txt:11]:
```suggestion
• Cerca i guiatge més ràpid i millorat
```

File: android/src/fdroid/play/listings/ca/release-notes.txt [hunk 12]
```diff
@@ -1,14 +1,13 @@
...
 
 …more: omaps.org/news
```
Comment (BennyBeat) [inline android/src/fdroid/play/listings/ca/release-notes.txt:13]:
```suggestion
…més informació: omaps.org/news
```

File: iphone/metadata/pt-BR/release_notes.txt [hunk 13]
```diff
@@ -1,27 +1,41 @@
...
+• Updated more than a year old Wikipedia articles
+• Fixed Wiki text in the dark mode
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia (map sizes increased)
+• More dense isolines in many areas
+• Reduced many other map sizes
+• Import comments from GPX cmt tag
+• Treat GPX track segments as part of the singular track they belong to
+• Fixed a bug when editing social contacts
+• Dog Park can be added in the Editor
+• Updated Persian voice directions
```
Comment (matheusgomesms) [inline iphone/metadata/pt-BR/release_notes.txt:11]:
```suggestion
• Novos dados OpenStreetMap de 14 de agosto
• Atualização de artigos da Wikipédia com mais de um ano
• Correção do texto da Wiki no modo escuro
• Adição de isolinhas ausentes para algumas regiões da Argentina, Chile, Cazaquistão, EUA, Rússia e Tibete (tamanho dos mapas aumentado)
• Isolinhas mais densas em muitas áreas
• Redução de muitos outros mapas
• Importação de comentários da etiqueta GPX cmt
• Tratamento dos segmentos de trilha GPX como parte da trilha singular à qual pertencem
• Correção de um bug ao editar contatos sociais
• Parques Caninos podem ser adicionado no Editor
• Atualização de direções de voz em persa
```

File: iphone/metadata/pt-PT/release_notes.txt [hunk 14]
```diff
@@ -1,27 +1,41 @@
...
+• Updated more than a year old Wikipedia articles
+• Fixed Wiki text in the dark mode
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia (map sizes increased)
+• More dense isolines in many areas
+• Reduced many other map sizes
+• Import comments from GPX cmt tag
+• Treat GPX track segments as part of the singular track they belong to
+• Fixed a bug when editing social contacts
+• Dog Park can be added in the Editor
+• Updated Persian voice directions
```
Comment (matheusgomesms) [inline iphone/metadata/pt-PT/release_notes.txt:11]:
```suggestion
• Novos dados OpenStreetMap de 14 de agosto
• Atualização de artigos da Wikipédia com mais de um ano
• Correção do texto da Wiki no modo escuro
• Adição de isolinhas ausentes para algumas regiões da Argentina, Chile, Cazaquistão, EUA, Rússia e Tibete (tamanho dos mapas aumentado)
• Isolinhas mais densas em muitas áreas
• Redução de muitos outros mapas
• Importação de comentários da etiqueta GPX cmt
• Tratamento dos segmentos de trilha GPX como parte da trilha singular à qual pertencem
• Correção de um bug ao editar contatos sociais
• Parques Caninos podem ser adicionado no Editor
• Atualização de direções de voz em persa
```

File: iphone/metadata/pt-BR/release_notes.txt [hunk 15]
```diff
@@ -1,27 +1,41 @@
...
 
-Busca
-• Melhoria na classificação dos resultados de pesquisa
-• Correção na busca de categorias de lojas
-• Correção nos resultados de pesquisa de aeroportos com nomes vazios
+Routing
+• Updated road surface factors for car routing profile
+• Updated pedestrian routing profile for Turkey and Georgia
+• Fixed cross-mwm routing when "avoid" options are enabled
```
Comment (matheusgomesms) [inline iphone/metadata/pt-BR/release_notes.txt:16]:
```suggestion
Roteamento
• Atualização de fatores de superfície da estrada para o roteamento de veículos
• Atualização de roteamento de pedestres para Turquia e Geórgia
• Correção do roteamento cross-mwm quando as opções "evitar" estão ativadas
```

File: iphone/metadata/pt-PT/release_notes.txt [hunk 16]
```diff
@@ -1,27 +1,41 @@
...
 
-Busca
-• Melhoria na classificação dos resultados de pesquisa
-• Correção na busca de categorias de lojas
-• Correção nos resultados de pesquisa de aeroportos com nomes vazios
+Routing
+• Updated road surface factors for car routing profile
+• Updated pedestrian routing profile for Turkey and Georgia
+• Fixed cross-mwm routing when "avoid" options are enabled
```
Comment (matheusgomesms) [inline iphone/metadata/pt-PT/release_notes.txt:16]:
```suggestion
Roteamento
• Atualização de fatores de superfície da estrada para o roteamento de veículos
• Atualização de roteamento de pedestres para Turquia e Geórgia
• Correção do roteamento cross-mwm quando as opções "evitar" estão ativadas
```

File: iphone/metadata/pt-PT/release_notes.txt [hunk 17]
```diff
@@ -1,27 +1,41 @@
...
-• Adição de centro comunitário, centro de conferências, centro de exposições, local de eventos
-• Aumento da prioridade dos ícones da loja
-• Adição de áreas protegidas
+Search
+• Fixed search for addresses in Czech and Slovakia
+• Improved street matching
+• Remove Tourist Information, Place of Worship, Park, Information Board, and Tourist Map from the Sightseeings category
+• Do not filter duplicated search results except of very close bus stops
+• Better ranking when searching by category
+• Fixed search by a house name
```
Comment (matheusgomesms) [inline iphone/metadata/pt-PT/release_notes.txt:24]:
```suggestion
Busca
• Correção de pesquisa de moradas na Eslováquia e República Checa
• Melhoria na correspondência de rua
• Remoção de Informações Turísticas, Local de Religião, Mapa Turístico, Painel de Informações e Parque da categoria Passeios Turísticos
• Não filtragem dos resultados de pesquisa duplicados, exceto para paragens de autocarro muito próximas
• Melhoria da classificação ao pesquisar por categoria
• Correção de busca por um nome de casa
```

File: iphone/metadata/pt-BR/release_notes.txt [hunk 18]
```diff
@@ -1,27 +1,41 @@
...
-• Adição de centro comunitário, centro de conferências, centro de exposições, local de eventos
-• Aumento da prioridade dos ícones da loja
-• Adição de áreas protegidas
+Search
+• Fixed search for addresses in Czech and Slovakia
+• Improved street matching
+• Remove Tourist Information, Place of Worship, Park, Information Board, and Tourist Map from the Sightseeings category
+• Do not filter duplicated search results except of very close bus stops
+• Better ranking when searching by category
+• Fixed search by a house name
```
Comment (matheusgomesms) [inline iphone/metadata/pt-BR/release_notes.txt:24]:
```suggestion
Busca
• Correção de pesquisa de endereços na Eslováquia e República Tcheca
• Melhoria na correspondência de rua
• Remoção de Informações Turísticas, Local de Religião, Mapa Turístico, Painel de Informações e Parque da categoria Passeios Turísticos
• Não filtragem dos resultados de pesquisa duplicados, exceto para pontos de ônibus muito próximos
• Melhoria da classificação ao pesquisar por categoria
• Correção de busca por um nome de casa
```

File: iphone/metadata/pt-BR/release_notes.txt [hunk 19]
```diff
@@ -1,27 +1,41 @@
...
+• Made bicycle parking icon differ more from the motorcycle one
+• Updated parking entrance icon
+• Street-side and lane parkings use a smaller icon and appear later when zooming in
+• Added a motorcycle parking and parking entrance icons to the navigation mode
+• Fixed motorway=trunk links visibility in navigation mode
+• Improved motorway=trunk colors
+• Removed too small isolines at overview zoom levels
+• Added Tehran metro icon
+• Revamped priorities system, especially for POIs
+• Many other fixes and enhancements
```
Comment (matheusgomesms) [inline iphone/metadata/pt-BR/release_notes.txt:39]:
```suggestion
Estilos
• Adição de rotas de balsa ao mapa do mundo
• Remoção de fundo quadrado dos ícones de estacionamento
• Adição de ícones distintos para estacionamentos pagos e privados
• Diferenciação do ícone de estacionamento de bicicletas em relação ao ícone de motocicleta
• Atualização do ícone de entrada de estacionamento
• Uso de ícone menor para estacionamentos ao lado da rua e faixas de estacionamento, aparecendo mais tarde ao aumentar o zoom
• Adição de ícones de estacionamento de motocicletas e de entrada de estacionamento ao modo de navegação
• Correção de visibilidade de ligação de autoestradas no modo de navegação
• Melhoria nas cores de autoestradas
• Remoção de isolinhas muito pequenas nos níveis de zoom da visão geral
• Adição de ícone do metrô de Teerã
• Reformulação de sistema de prioridades, especialmente para POIs
• Muitas outras correções e melhorias
```

File: iphone/metadata/pt-PT/release_notes.txt [hunk 20]
```diff
@@ -1,27 +1,41 @@
...
+• Made bicycle parking icon differ more from the motorcycle one
+• Updated parking entrance icon
+• Street-side and lane parkings use a smaller icon and appear later when zooming in
+• Added a motorcycle parking and parking entrance icons to the navigation mode
+• Fixed motorway=trunk links visibility in navigation mode
+• Improved motorway=trunk colors
+• Removed too small isolines at overview zoom levels
+• Added Tehran metro icon
+• Revamped priorities system, especially for POIs
+• Many other fixes and enhancements
```
Comment (matheusgomesms) [inline iphone/metadata/pt-PT/release_notes.txt:39]:
```suggestion
Estilos
• Adição de rotas de balsa ao mapa do mundo
• Remoção de fundo quadrado dos ícones de estacionamento
• Adição de ícones distintos para estacionamentos pagos e privados
• Diferenciação do ícone de estacionamento de bicicletas em relação ao ícone de motocicleta
• Atualização do ícone de entrada de estacionamento
• Uso de ícone menor para estacionamentos ao lado da rua e faixas de estacionamento, aparecendo mais tarde ao aumentar o zoom
• Adição de ícones de estacionamento de motocicletas e de entrada de estacionamento ao modo de navegação
• Correção de visibilidade de ligação de autoestradas no modo de navegação
• Melhoria nas cores de autoestradas
• Remoção de isolinhas muito pequenas nos níveis de zoom da visão geral
• Adição de ícone do metrô de Teerão
• Reformulação de sistema de prioridades, especialmente para POIs
• Muitas outras correções e melhorias
```

File: iphone/metadata/pt-PT/release_notes.txt [hunk 21]
```diff
@@ -1,27 +1,41 @@
...
 
-Acompanhe nossas novidades em https://omaps.org/news
+Follow our news at https://omaps.org/news
```
Comment (matheusgomesms) [inline iphone/metadata/pt-PT/release_notes.txt:41]:
```suggestion
Acompanhe nossas novidades em https://omaps.org/news
```

File: iphone/metadata/pt-BR/release_notes.txt [hunk 22]
```diff
@@ -1,27 +1,41 @@
...
 
-Acompanhe nossas novidades em https://omaps.org/news
+Follow our news at https://omaps.org/news
```
Comment (matheusgomesms) [inline iphone/metadata/pt-BR/release_notes.txt:41]:
```suggestion
Acompanhe nossas novidades em https://omaps.org/news
```

**Review by matheusgomesms — CHANGES_REQUESTED** [review]
PT and PT-BR reviewed

**Comment by 501Ghost** [comment]
Dutch (NL) translation. It took me some head scratching to get it under the 500 character limit. I hope no one has beaten me to it.
```
• Nieuwe OpenStreetMap-gegevens van 14 augustus
• Verse Wikipedia-artikelen
• Fix zoeken naar Tsjechische en Slowaakse adressen
• Contourlijnen toegevoegd aan regio's van Argentinië, Chili, Kazachstan, Tibet, VS, Rusland
• Hogere dichtheid van contourlijnen in sommige plaatsen
• Bugfix voor Nexus 5X
• Veerroutes toegevoegd aan de wereldkaart
• Nieuwe parkeerplaats-iconen
• Prioriteitensysteem voor POI's vernieuwd
• Kleine verbeteringen en bugfixes
• Sneller, beter zoeken en routeren

…meer: omaps.org/news
```
I'm on holiday and I can't be bothered to figure out how to commit things properly (I'm a noob), so I would be happy if someone could do this for me.

File: android/src/fdroid/play/listings/ar/release-notes.txt [hunk 23]
```diff
@@ -1,14 +1,13 @@
...
-• تمت إضافة المتاجر الجديدة وأماكن الفعاليات
-• تحسين أنفاق وجسور المياه
-• سكك حديدية أكثر تفصيلا
-• الحدود المضافة=المنطقة_المحمية
+• New OpenStreetMap data as of August 14
+• Fresh Wikipedia articles
+• Fixed search for addresses in Czech and Slovakia
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia
+• More dense isolines in some areas
+• Fixed bug on Nexus 5X
```
Comment (rtsisyk) [inline android/src/fdroid/play/listings/ar/release-notes.txt:6]: What bug?

File: android/src/fdroid/play/listings/ar/release-notes.txt [hunk 24]
```diff
@@ -1,14 +1,13 @@
...
+• Added ferry routes to the World map
+• Redesigned parking icons
+• Revamped priorities system for POIs
```
Comment (rtsisyk) [inline android/src/fdroid/play/listings/ar/release-notes.txt:9]: What does it mean?

File: android/src/fdroid/play/listings/ar/release-notes.txt [hunk 25]
```diff
@@ -1,14 +1,13 @@
...
+• Many other smaller enhancements and bug fixes
+• Faster, improved search and routing
```
Comment (rtsisyk) [inline android/src/fdroid/play/listings/ar/release-notes.txt:11]: Any details?

File: android/src/fdroid/play/listings/be/release-notes.txt [hunk 26]
```diff
@@ -1,12 +1,12 @@
...
-• Выпраўлена некалькі збояў
-• Падтрымка вышыні, OsmAnd і колераў Garmin у імпартаваных дарожках GPX
-• Выпраўлены час прыбыцця для пешага маршрута з вялікімі схіламі
-• Кнопка «Абагуліць» была перамешчана на старонку «Месца»
-• Абноўлены пераклады
-• Даданы новыя крамы і тыпы мерапрыемстваў
-• Палепшаны рэндэрынг водных тунэляў і мастоў
-• Больш падрабязныя тыпы чыгункі
-• Даданая boundary=protected_area
+• Новыя дадзеныя OpenStreetMap па стане на 14 жніўня
```
Comment (zubialevich) [inline android/src/fdroid/play/listings/be/release-notes.txt:1]:
```suggestion
• Новыя даныя OpenStreetMap па стане на 14 жніўня
```

File: android/src/fdroid/play/listings/be/release-notes.txt [hunk 27]
```diff
@@ -1,12 +1,12 @@
...
+• Свежыя артыкулы Вікіпедыі
```
Comment (zubialevich) [inline android/src/fdroid/play/listings/be/release-notes.txt:2]:
```suggestion
• Свежыя артыкулы з Вікіпедыі
```

File: android/src/fdroid/play/listings/uk/release-notes.txt [hunk 28]
```diff
@@ -1,11 +1,13 @@
...
+• Fresh Wikipedia articles
+• Fixed search for addresses in Czech and Slovakia
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia
+• More dense isolines in some areas
+• Fixed bug on Nexus 5X
+• Added ferry routes to the World map
+• Redesigned parking icons
+• Revamped priorities system for POIs
+• Many other smaller enhancements and bug fixes
+• Faster, improved search and routing
```
Comment (strump) [inline android/src/fdroid/play/listings/uk/release-notes.txt:11]:
```suggestion
• Нові дані OSM станом на 14 серпня
• Свіжі статті з Вікіпедії для ...
• Виправлення адресного пошуку в Чехії та Словакії
• Додано ізолінії в деяких регіонах Аргентини, Чилі, Казахстану, Тібету, США, Росії
• Підвищено щільність ізоліній в деяких регіонах
• Виправлення помилки на Nexus 5X
• Паромні переправи відображаються на оглядовій мапі Світу
• Оновлений дизайн іконок паркінгів
• Перероблена система приоритетів для POI
• Купа маленьких поліпшень та виправлень помилок
• Пошуку та будування маршрутів стали швидше та точніше
```

File: android/src/fdroid/play/listings/uk/release-notes.txt [hunk 29]
```diff
@@ -1,11 +1,13 @@
...
 
-https://omaps.org/news
+…more: omaps.org/news
```
Comment (strump) [inline android/src/fdroid/play/listings/uk/release-notes.txt:13]:
```suggestion
...більше: omaps.org/news
```

**Review by strump — COMMENTED** [review]
Added Ukrainian translation

**Comment by alexgabi** [comment]
BASQUE TRANSLATION (eu)

• OpenStreetMap-en datu berriak abuztuaren 14tik aurrera
• Wikipediako artikulu berriak
• Txekia eta Eslovakiako helbideen konponketa
• Argentina, Txile, Kazakhstan, Tibet, AEB, Errusia eskualde batzuetarako falta diren isolineak gehitu dira
• Zona batzuetan isolinea trinkoagoak
• Nexus 5X-n akatsa konpondu da
• Ferry-bideak gehitu dira Munduko mapan
• Aparkalekuen ikonoak birmoldatuak
• Interesgune edo POIentzako lehentasunen sistema berritua
• Beste hobekuntza txikiago asko eta akatsen konponketa
• Bilaketa eta bideratze azkarragoa eta hobetua

**Comment by baldrianbandit** [comment]
de_DE
• Neue Openstreetmap-Daten mit Stand 14. August
• Aktuelle Wikipedia-Artikel
• Suche von Adressen funktioniert nun auch für Tschechien und Slovakei
• Fehlende Konturlinien für einige Regionen in Argentinien, Chile, Kasachstan, Tibet, USA, Russland hinzugefügt
• Bessere Konturlinien für einige Regionen
• Fehler bei Nexus 5X behoben
• Fährverbindungen zur Weltkarte hinzugefügt
• Überarbeitete Symbole für Parkplätze
• Verbesserte Priorisierung von Sehenwürdigkeiten
• Viele kleine Verbesserungen und Fehlerbehebungen
• Schnellere und verbesserte Suche und Routenführung

File: android/src/fdroid/play/listings/it-IT/release-notes.txt [hunk 30]
```diff
@@ -1,14 +1,13 @@
...
+• Fresh Wikipedia articles
+• Fixed search for addresses in Czech and Slovakia
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia
+• More dense isolines in some areas
+• Fixed bug on Nexus 5X
+• Added ferry routes to the World map
+• Redesigned parking icons
+• Revamped priorities system for POIs
+• Many other smaller enhancements and bug fixes
+• Faster, improved search and routing
```
Comment (rayz1065) [inline android/src/fdroid/play/listings/it-IT/release-notes.txt:11]: • Nuovi dati di OpenStreetMap aggiornati al 14 agosto
• Aggiornati articoli di Wikipedia
• Aggiustata la ricerca per indirizzi in Cechia e Slovacchia
• Aggiunte linee isometriche mancanti per alcune regioni di Argentina, Cile, Kazakistan, Tibet, USA, Russia
• Rese linee isometriche più dense in alcune aree
• Risolto bug su Nexus 5X
• Aggiunte rotte dei traghetti alla mappa del mondo
• Ridisegnate icone dei parcheggi
• Ammodernato sistema di priorità per i luoghi d'interesse
• Tanti altri piccoli miglioramenti e bugfix
• Migliorata velocità di ricerca e indicazioni stradali

**Review by rayz1065 — CHANGES_REQUESTED** [review]

File: android/src/fdroid/play/listings/it-IT/release-notes.txt (hunk 30 above)
Comment (rayz1065) [inline android/src/fdroid/play/listings/it-IT/release-notes.txt:11]:
```suggestion
• Nuovi dati di OpenStreetMap aggiornati al 14 agosto
• Aggiornati articoli di Wikipedia
• Aggiustata la ricerca per indirizzi in Cechia e Slovacchia
• Aggiunte linee isometriche mancanti per alcune regioni di Argentina, Cile, Kazakistan, Tibet, USA, Russia
• Rese linee isometriche più dense in alcune aree
• Risolto bug su Nexus 5X
• Aggiunte rotte dei traghetti alla mappa del mondo
• Ridisegnate icone dei parcheggi
• Ammodernato sistema di priorità per i luoghi d'interesse
• Tanti altri piccoli miglioramenti e bugfix
• Migliorata velocità di ricerca e indicazioni stradali
```

Comment (rayz1065) [inline android/src/fdroid/play/listings/it-IT/release-notes.txt:11]:
```suggestion
• Nuovi dati di OpenStreetMap aggiornati al 14 agosto
• Aggiornati articoli di Wikipedia
• Aggiustata la ricerca per indirizzi in Cechia e Slovacchia
• Aggiunte linee isometriche mancanti per alcune regioni di Argentina, Cile, Kazakistan, Tibet, USA, Russia
• Rese linee isometriche più dense in alcune aree
• Risolto bug su Nexus 5X
• Aggiunte rotte dei traghetti alla mappa del mondo
• Ridisegnate icone dei parcheggi
• Ammodernato sistema di priorità per i luoghi d'interesse
• Tanti altri piccoli miglioramenti e bugfix
• Migliorata velocità di ricerca e indicazioni stradali
```

**Review by rayz1065 — CHANGES_REQUESTED** [review]

**Comment by d4f5409d** [comment]
### Translation for hu 🇭🇺

```- Új OpenStreetMap adatok (augusztus 14)
 - Friss Wikipédia cikkek 
 - Javítottuk a címek keresését Csehországban és Szlovákiában 
 - Hozzáadtuk Argentína, Chile, Kazahsztán, Tibet, USA, Oroszország egyes régióinak hiányzó határolóvonalait. 
 - Sűrűbb határolóvonalak néhány területen 
 - Javítottuk a hibát a Nexus 5X készülékeken 
 - Hozzáadtuk a kompútvonalak a világtérképhez 
 - Újraterveztük a parkolási ikonokat 
 - Módosítottunk a POI-k megjelenítésének prioritási rendszerén 
 - Sok más kisebb fejlesztés és hibajavítás 
 - Gyorsabb, pontosabb keresés és útvonaltervezés 
 
 
  ...még több: omaps.org/news```

**Comment by d4f5409d** [comment]
Please consider replacing 1 line with "Please support us with a donation!"

File: iphone/metadata/pl/release_notes.txt [hunk 31]
```diff
@@ -1,30 +1,41 @@
...
-• Prefer tertiary roads to service roads in car routing
+• New OpenStreetMap maps data as of August 14
+• Updated more than a year old Wikipedia articles
+• Fixed Wiki text in the dark mode
+• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia (map sizes increased)
+• More dense isolines in many areas
+• Reduced many other map sizes
+• Import comments from GPX cmt tag
+• Treat GPX track segments as part of the singular track they belong to
+• Fixed a bug when editing social contacts
```
Comment (baq-86) [inline iphone/metadata/pl/release_notes.txt:9]:
```suggestion
• Naprawiony błąd podczas edycji kontaktów społecznościowych
```

File: iphone/metadata/pl/release_notes.txt [hunk 32]
```diff
@@ -1,30 +1,41 @@
...
+• Dog Park can be added in the Editor
```
Comment (baq-86) [inline iphone/metadata/pl/release_notes.txt:10]:
```suggestion
• Edytor umożliwia dodawanie psich parków
```

File: iphone/metadata/pl/release_notes.txt [hunk 33]
```diff
@@ -1,30 +1,41 @@
...
+• Updated Persian voice directions
```
Comment (baq-86) [inline iphone/metadata/pl/release_notes.txt:11]:
```suggestion
• Zaktualizowane perskie wskazówki głosowe
```

**Review by baq-86 — CHANGES_REQUESTED** [review]
My suggestions for pl-PL - all translated for android, for iphone - partially translated. I will finish it a bit later...

File: android/src/fdroid/play/listings/ar/release-notes.txt (hunk 23 above)
Comment (biodranik) [reply android/src/fdroid/play/listings/ar/release-notes.txt:6]: Owners know it. OM doesn't work.

File: android/src/fdroid/play/listings/ar/release-notes.txt (hunk 25 above)
Comment (biodranik) [reply android/src/fdroid/play/listings/ar/release-notes.txt:11]: Yes, in a full news post, not limited by 500 characters policy.

File: android/src/fdroid/play/listings/be/release-notes.txt (hunk 26 above)
Comment (biodranik) [reply android/src/fdroid/play/listings/be/release-notes.txt:1]: Should we fix it also in the About dialog?

File: android/src/fdroid/play/listings/ar/release-notes.txt (hunk 24 above)
Comment (biodranik) [reply android/src/fdroid/play/listings/ar/release-notes.txt:9]: Detailed news without 500 chars limit explains it.

**Comment by biodranik** [comment]
> Please consider replacing 1 line with "Please support us with a donation!" We could include this in every releases' notes.

That won't work for Google Play and App Store.

**Comment by biodranik** [comment]
Thank you! Android translations can be updated later if necessary.

**Comment by d4f5409d** [comment]
> > Please consider replacing 1 line with "Please support us with a donation!" We could include this in every releases' notes.
> 
> That won't work for Google Play and App Store.

I see.

**Review by baq-86 — COMMENTED** [review]
A few more translated lines in pl-PL (iPhone)... More to come :)

**Pull request merged**